web-client/src/presenter/presenter.ts

```diff
@@ -1490,6 +1491,7 @@ export const presenterSequences = {
     validateChangeLoginAndServiceEmailSequence as unknown as Function,
   validateCourtIssuedDocketEntrySequence:
     validateCourtIssuedDocketEntrySequence as unknown as Function,
+  validateCourtOrderSequence: validateCourtOrderSequence as unknown as Function,
```

Comment:
We should only be doing as unknown as Function for old Sequences. All new sequences should be typed to indicate what their props are.

docs/typesOnTSX.md